escalation/store.go

```diff
@@ -286,6 +287,22 @@ func (s *Store) _updateStepTarget(ctx context.Context, stepID string, tgt assign
 	return err
 }
 
+func (s *Store) newWebhook(ctx context.Context, tx *sql.Tx, webhookTarget assignment.Target) (assignment.Target, error) {
```

Comment:
nit: call this `chanWebhook` since it doesn't always create a new one

second: add a check here to reject with a validation error if the experimental flag isn't set. See `contactmethod.go:89` for an example